### ISSUE TITLE
Bug: api.php sometimes returns "Warning: Cannot modify header information - headers already sent..."

### ISSUE THREAD

**Issue by pskillen** [NanoTools/nanoNodeMonitor#158]
## Summary

Very intermittently, api.php returns some warnings before the JSON response. The JSON response does appear to be well formed otherwise.

This is quite hard to reproduce in a browser (many, many refreshes required). However I'm trying to call the API from Zabbix (monitoring software) and the error happens >90% of the time. Zabbix is polling the API every 3 mins

## Environment

* Nano Node Monitor v1.6.1
* Official Docker image
* Both for Nano node and Banano node

## Error data

```
Warning: Undefined property: stdClass::$store_vendor in /var/www/html/api.php on line 46

Warning: Undefined property: stdClass::$block_count in /var/www/html/api.php on line 181

Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/api.php:46) in /var/www/html/modules/functions.php on line 92

Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/api.php:46) in /var/www/html/modules/functions.php on line 93
{"nanoNodeAccount":"ban_3paddy7uu11k6razie9afqkfkqtuphxnhyigojiqmqesa73p6p1hwtiz8uxi","nanoNodeAccountShort":"ban_3padd...z8uxi","nanoNodeAccountUrl":"https:\/\/creeper.banano.cc\/explorer\/account\/ban_3paddy7uu11k6razie9afqkfkqtuphxnhyigojiqmqesa73p6p1hwtiz8uxi","version":"BANANO 
...
```

**Comment by BitDesert** [comment]
Do you use the Docker image or your own PHP implementation?
Is anything else running at the same moment as the check on the node?
What happens when you diable the caching?

**Comment by pskillen** [comment]
Sorry, left out that very vital piece of info. Yes, the official docker image (`nanotools/nanonodemonitor:v1.6.1`). It's running on a server with a reasonably heavy load (nano node, banano node, nano monitor, banano monitor, some other stuff - all dockerised).

I've not tried disabling caching yet - not aware how to do that

**Comment by BitDesert** [comment]
Some occurences should be fixed with 0768dc935fe344dd6a3044241665c1cd6ab31d7c
